<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPrestamo</class>
 <widget class="QWidget" name="FormPrestamo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblCota">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cota: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCota">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTipoPrestamo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tipo Préstamo: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTipoPrestamo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Aula</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hogar</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditAutor">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTitulo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCedula">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblAutor">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Autor: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitulo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Título: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCedula">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cédula: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFechaP">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fecha Préstamo: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFechaE">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fecha Entrega: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblResponsable">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Responsable: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFechaP">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>El formato de la fecha es: DD/MM/AAAA</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="lineEditFechaE">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>El formato de la fecha es: DD/MM/AAAA</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxResponsable">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>01</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>02</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnPrestar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Prestar</string>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/images/book.png</normaloff>:/images/book.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
